core/domain/state_domain.py

```diff
@@ -797,7 +797,8 @@ class WrittenTranslations(python_utils.OBJECT):
     """
 
     def __init__(self, translations_mapping):
-        """Initializes a WrittenTranslations domain object."""
+        """Initializes a WrittenTranslations domain object.
+        """
```

Comment:
Add the `Args` docstring here

Reply:
I'm not sure why @kevinlee12 resolved this -- it does not seem to be addressed. Unresolving.

Reply:
oops

Reply:
Hmm, I think I may have intended to resolve another comment.